src/UI/Badge.elm

```diff
@@ -195,88 +235,159 @@ successDark content =
 
 -}
 withTone : (String -> Badge) -> Badge -> Badge
-withTone builder (Badge { content }) =
+withTone builder (Badge { content } _) =
```

Comment:
This will discard the options, and we don't want that, so here we'll need a more complex manoeuvre, like:
```suggestion
withTone builder (Badge { content } originalOptions) =
   let (Badge newProps _) = builder content
   in (Badge newProps originalOptions)
```